### ISSUE TITLE
Bug: `prefer-node-protocol` does not report `https`

### ISSUE THREAD

**Issue by ota-meshi** [eslint-community/eslint-plugin-n#431]
### Environment

Node version: 22.12.0
npm version: 10.9.0
ESLint version: 9.23.0
eslint-plugin-n version: 17.17.0
Operating System: MacOS

### What rule do you want to report?

prefer-node-protocol

### Link to Minimal Reproducible Example

https://eslint-online-playground.netlify.app/#eNp9UMtOwzAQ/BVrL7k0TiMhkIJAHPgA7nWRInsTUvyS7aCiNv+O7SQocOjFiXdmZzxzAe94hedWWYn05KGBQVnjAvkIwXrSOaMIg3xh8Mj0Bt2CGYMdoJeDDpQb3Q39Hz1tBL7JsR/0ujdzS5uHpc4SeM5kgV07ykAOTJPN5qLrDww62YbKITdKoRYoSmXEKJHBcZd2LukgxMWRb9YbIYWurMMOXZlES+tMMNzIoiEFOmdcMROn9InHcQ5lW/7Z9qkeo2OgrMYgfNto18S/1To7x1w6PhVjXb/GDJLdTH6v72j9QPfkeiXPT/U93dM9g2y47Av8ekWbUmk+/NN5mUurTnnMILaAPizW2TzjN8FN41vWknmC6Qc87LAV

### What did you expect to happen?

I would expect the following code to report both `https` and `http`, but it only reports `http`.

```js
import https from "https";
import http from "http";
```

### Participation

- [x] I am willing to submit a pull request for this issue.
